generators/app/index.js

```diff
@@ -364,53 +364,9 @@ module.exports = class extends BaseGenerator {
                 this.configOptions.clientPackageManager = this.clientPackageManager;
             },
 
-            composeServer() {
-                if (this.skipServer) return;
-                const options = this.options;
-                const configOptions = this.configOptions;
-
-                this.composeWith(require.resolve('../server'), {
-                    ...options,
-                    configOptions,
-                    'client-hook': !this.skipClient,
-                    debug: this.isDebugEnabled
-                });
-            },
-
-            composeClient() {
-                if (this.skipClient) return;
-                const options = this.options;
-                const configOptions = this.configOptions;
-
-                this.composeWith(require.resolve('../client'), {
-                    ...options,
-                    configOptions,
-                    debug: this.isDebugEnabled
-                });
-            },
-
-            composeCommon() {
-                const options = this.options;
-                const configOptions = this.configOptions;
-
-                this.composeWith(require.resolve('../common'), {
-                    ...options,
-                    'client-hook': !this.skipClient,
-                    configOptions,
-                    debug: this.isDebugEnabled
-                });
-            },
-
-            askFori18n: prompts.askFori18n
-        };
-    }
-
-    get default() {
-        return {
+            askFori18n: prompts.askFori18n,
```

Comment:
Doesn't this depend on value from the client?

Reply:
askFori18n shouldn't depend on value from client.

common/client/server modules depends on each other, but at default step, every module have configuration in place.

That the reason of https://github.com/mshima/generator-jhipster/commit/25c469bf7ee12a5538050ba8919a75a2c7823105

If you have doubt about yeoman workflow, take a look at the log from azure project step.
I've enabled yeoman debug.
And you can take a look at https://github.com/yeoman/generator/blob/2202fab7231707aeabbc2461d41e6bcafc743524/test/generators-compose-workflow.js